### DISCUSSION TITLE
Cannot Push my submission

### DISCUSSION THREAD

**Discussion by goofynugtz** [DiveHQ/backend-internship-task#26] [Q&A]
I'm not able to clone or fork my private repository as it says forking is disabled. 
When I try to clone the private repo by ssh, it throws an ERROR: Repository not found.
![Screenshot from 2023-06-18 13-17-17](https://github.com/DiveHQ/backend-internship-task/assets/75770585/c9118c0b-7f04-47f4-b502-cc90138bef39)

For reference, I'm able to clone another private repo by ssh.
SInce I'm not able to clone the repo, all I'm able to do is drag and drop to submit on the web browser, in which I'm not able to drop dotfiles. Also It becomes a giant single commit.

I've even tried setting the remote origin url but that shows the same error.

How do I submit all my previous commits?

**Comment by prayashm** [comment]
Are you sure that you are pushing as this user account (it has access):
<img width="381" alt="image" src="https://github.com/DiveHQ/backend-internship-task/assets/424487/ccf9d4b3-dedc-4508-99ff-626c2375a873">

**Reply by goofynugtz** [reply]
Sorry, I didn't leave an update here.
I figured out a way to push to the repo through GitHub Classroom extension for VSCode. 
I did create a PR before 6PM deadline, and later on merged it.
Can you confirm, will it be considered for submission? @prayashm
